pkg/stubs/network_stub.go

```diff
@@ -328,7 +329,7 @@ func (s *NetworkerStub) RemovePubTap(ctx context.Context, arg0 string) (ret0 err
 	return
 }
 
-func (s *NetworkerStub) RemoveTap(ctx context.Context, arg0 zos.NetID) (ret0 error) {
+func (s *NetworkerStub) RemoveTap(ctx context.Context, arg0 string) (ret0 error) {
```

Comment:
By the way, we have a tool to generate stubs to match the interface from pkg/. Do u change this manually ?

Reply:
Install it like this 
```
go install github.com/threefoldtech/zbus/zbusc
```
make sure now `zbusc` in your path. 
then do 
```
cd zos/pkg
go generate ./...
```

The generate will use the zbus tool to generate the stubs.